### PR TITLE
new example project

### PR DESCRIPTION
**Pull request by venosov**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -612,3 +612,4 @@ If you are willing to contribute please don't hesitate to discuss in issues and
 | https://github.com/stchar/pipeline-sharedlib-testharness          | Junit, Gradle(Groovy) | SharedLibrary | Integration, Unit |
 | https://github.com/stchar/pipeline-dsl-seed                       | Junit, Spock, Gradle(Groovy) | scripted pipeline | Integration(jobdsl), Unit |
 | https://github.com/SpencerMalone/JenkinsPipelineIntegration       | Spock, Gradle(Groovy) | SharedLibrary | Integration |
+| https://github.com/venosov/jenkins-pipeline-shared-library-example-victor       | Junit, Gradle(Kotlin) | SharedLibrary | Unit |
```
Comment (stchar) [inline README.md]: You have gradle.kts example this is cool.
You're mentioning that it has shared library examples but there are none (at least at version https://github.com/venosov/jenkins-pipeline-shared-library-example-victor/commit/eacd9913d5f2a090369eee94851d318015bbf4eb)

Comment (venosov) [reply README.md]: Hi, here you have https://github.com/venosov/jenkins-pipeline-shared-library-example-victor/blob/master/pipelineTests/groovy/BuildPipelineTest.groovy

Comment (stchar) [reply README.md]: Hi,
The test you shared is a scripted pipeline test. 

I expected to see something like https://github.com/jenkinsci/JenkinsPipelineUnit/blob/master/README.md#testing-shared-libraries

Comment (venosov) [reply README.md]: Thanks, I see your point, here's the new commit https://github.com/venosov/jenkins-pipeline-shared-library-example-victor/commit/be29c9c4bfd26a49444b7cb0f999710bc6ca02f9

**Comment by stchar** [comment]
Cool, thanks, merged

**Pull request merged**